src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exceptions.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static User user;
+    private static StorageDao storageDao;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        user = new User();
+        storageDao = new StorageDaoImpl();
+    }
+
+    @Test
+    void userAgeLessThenNeed_NotOk() {
+        user.setAge(12);
+        assertThrows(RegistrationException.class, () -> registrationService.register(user));
```

Comment:
<img width="782" alt="image" src="https://user-images.githubusercontent.com/81781479/210868376-02beee94-07f4-4fb2-a0d4-098e6dbbd1e4.png">

add also access modifiers to all methods